### PR TITLE
Pre-prod.

### PR DESCRIPTION
**Pull request by dkorolev**
Hi @mzhurovich 

Should be good to go modulo the last remaining notifications.

All tests pass upon re-generating `cards.json` using `./.current/convert_cards` with the top five lines of `cards.txt` removed (otherwise the starting cards return atop the real ones, even the user-added ones).

Pushing `cards.txt` into the repo blessed by GL.

Thanks,
Dima

### CODE REVIEW

**Comment by dkorolev** [comment]
@mzhurovich -- had to change the hash for the color, since the first two intro cards have been the same. PTAL!

File: server.h [hunk 1]
```diff
@@ -129,13 +127,22 @@ class CTFOServer final {
     event_log_stream_.open(event_log_file_, std::ofstream::out | std::ofstream::app);
 
     bricks::cerealize::CerealFileParser<Card, bricks::cerealize::CerealFormat::JSON> cf(cards_file);
+    const UID admin_uid = static_cast<UID>(1000000000000000001ull);
```
Comment (mzhurovich) [inline server.h:130]: I suppose, admin user could be used in many places, so maybe define it more globally?

**Comment by mzhurovich** [comment]
@dkorolev Смержу для ясности :)

**Comment by dkorolev** [comment]
Только сел фикс сделать :-)

**Pull request merged**